dev-util/webstorm/webstorm-2017.2.2.172.3757.55.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils versionator
+
+SLOT="$(get_major_version)"
```

Comment:
Please put it in its rightful place, and keep variables ordered/grouped as in devmanual/skel, i.e.:

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE

...